lib/ipmi_strings.c

```diff
@@ -830,15 +830,15 @@ int ipmi_oem_info_init()
 	if ((home = getenv("HOME"))) {
 		char path[PATH_MAX];
 		strncpy(path, home, sizeof(path));
-		strcat(path, "/.local" IANA_PEN_REGISTRY);
+		strcat(path, "/" IANAUSERDIR "/" IANA_PEN_REGISTRY);
 		in = fopen(path, "r");
 	}
 
 	if (!in) {
 		/*
 		 * Now open the system default file
 		 */
-		in = fopen(IANA_PEN_REGISTRY, "r");
+		in = fopen(IANADIR "/" IANA_PEN_REGISTRY, "r");
```

Comment:
I was of the impression some users were running this on windows?  Perhaps just with cygwin.

Reply:
Yes, so far ipmitool doesn't build with mingw32, only with cygwin.
I personally haven't built it (or anything else) for windows even with cygwin, so I just don't know how cygwin treats paths.

Anyway, if you know a portable way of building this path, I'd be glad to see it. I'll try to investigate that myself too...

Reply:
I don't build for windows myself.  I found this:

const char kPathSeparator =
#ifdef _WIN32
                            '\\';
#else
                            '/';
#endif

https://stackoverflow.com/questions/12971499/how-to-get-the-file-separator-symbol-in-standard-c-c-or

Honestly, it's not something you need to fix, but it's something I wanted to ask about as I know some people compile this.  If mingw32 works with "/" fine then this is fine.

Reply:
I believe I found the way. See the updated patch set.